### DISCUSSION TITLE
Files placed in public - Documents directory in web server does not show in browser

### DISCUSSION THREAD

**Discussion by rocheston** [DustinBrett/daedalOS#233] [Q&A] [answered]
I exported the project using yarn export and published them on my web server. I placed some files in Documents directory under users public. Nothing shows up when I navigate in file explorer in web browser.

![grab0](https://user-images.githubusercontent.com/16910669/202651512-2a4f0eb7-82d0-4367-b153-e47fb98432b9.jpg)

**Comment by DustinBrett** [comment, marked as answer]
It requires running the build to update the JSON file.

You can also just run `yarn build:fs`, after updating files in `public`. Also, the folder you are showing is `/Users/Public`, which is actually in the real file system at `public/Users/Public`. It's a bit confusing but Next.js uses the same wording.

**Comment by rocheston** [comment]
That works. Thank you Dustin. Great project. I love daedalOS  

Another question. How do I change the default wallpaper with my own wallpaper jpg. Use my wallpaper when the OS loads.

**Comment by DustinBrett** [comment]
Glad it worked!

To store the wallpaper setting you would also need to update the default [session.json](https://github.com/DustinBrett/daedalOS/blob/main/public/session.json) to have the wallpaper settings you want. Ex:

```json
{
  "wallpaperFit": "fill",
  "wallpaperImage": "/Users/Public/Pictures/Dustin/My Eye.jpg",
}
```

**Comment by rocheston** [comment]
This works. Thank you Sir!  

![grab1](https://user-images.githubusercontent.com/16910669/202910890-5a78b3f9-00e7-4751-8b5f-c3f282bf5633.jpg)
